pkg/rigfs/righelper.sh

```diff
@@ -88,7 +87,8 @@ main() {
       fi
       echo -n "{\"dir\":["
       first=true
-      for file in "$path"/*; do
+      for file in "$path"/.* "$path"/*; do
+        case "$file" in "$path"/. | "$path"/..) continue ;; esac
```

Comment:
I think `.` and `..` need to be there

Reply:
Do you mean that those paths should be returned to rig? I don't think so. I think I remember that the tests were failing without skipping those.

This is basically the POSIX workaround for `shopt -s dotglob`. The bash docs also state this:

> The filenames ‘.’ and ‘..’ must always be matched explicitly, even if dotglob is set.

So even with dotglob, bash will suppress those.

Reply:
Yeah but this is for the fs.FS support (which is pretty much unused currently), for things like fs.WalkDir, fs.ReadDir etc.

Reply:
When `.` and `..` aren't omitted, the tests enter an endless loop, because they'll recurse into `.`. So I think this is required.

> (which is pretty much unused currently)

k0sctl uses it, which means that it won't work on stock Alpine at the moment, it requires `apk add coreutils bash` and `apk add sudo && apk del doas`, but the latter is another issue.